Lexicon.md

```diff
@@ -4,19 +4,20 @@
-* __Agent Boundary__: The surface that separates an __Agent__ from its __Environment__.
-* __Observation__: __Information__ that crosses the __Agent Boundary__, entering the __Agent__.
-* __Observation Space__: The set of all __Observations__ that a particular __Agent__ may observe. All __Observations__ perceived by an __Agent__ are sampled from its __Observation Space__.
+* __Actor__: An __Object__ that interacts with an __Environment__ through __Events__. The difference between an __Actor__ and an __Object__ lies in its relevancy to a particular __Event__ (__Actors__ being relevant). The interpretation of the __Event__ crystallizes what is considered an __Actor__.
+* __Actor Boundary__: The surface that separates an __Actor__ from its __Environment__.
+* __Event__: __Information__ that crosses the __Actor Boundary__.
+* __Observation__: An __Event__ in which __Information__ enters an Agent.
+* __Observation Space__: The set of all __Observations__ that a particular Agent may observe. All __Observations__ perceived by an Agent are sampled from its __Observation Space__.
 * __Object__: A set of correlated __Observations__.
 * __System__: An __Object__ composed of correlated member __Objects__.
 * __Model__: A __System__ that represents another __System__. It can be assigned a truth value (measurement) which expresses the accuracy of the representation. Examples of __Models__ include maps, blueprints, simulations, mathematical models, beliefs and claims.
 * __World State__: Given that the __Information__ in an __Environment__ may change with respect to some parameter (for example, time), __World State__ defines a static snapshot of the __Information__ within an __Environment__. In other words, a __World State__ is an __Environment__ where all __Information__ is held constant.
-* __World Model__: An __Agent__'s internal __Model__ of its __World__.
-* __Action__: __Information__ that crosses the __Agent Boundary__, exiting the __Agent__.
-* __Action Space__: The set of all __Actions__ that a particular __Agent__ may perform. All __Actions__ performed by an __Agent__ are sampled from its __Action Space__.
-* __Value__: An __Agent__'s expression of preference between two __World States__. __Value__ may favour either __State__, or may reflect indifference. __Value__ is not intrinsic, and cannot be specified in the absence of an __Agent__.
+* __World Model__: An Agent's internal __Model__ of its __World__. 
+* __Action__: An __Event__ in which __Information__ exits an Agent.
+* __Action Space__: The set of all __Actions__ that a particular Agent may perform. All __Actions__ performed by an Agent are sampled from its __Action Space__.
+* __Value__: An Agent's expression of preference between two __World States__. __Value__ may favour either __State__, or may reflect indifference. __Value__ is not intrinsic, and cannot be specified in the absence of an __Actor__.
```

Comment:
... in the absence of an Agent.

Reply:
Yeah, I mentioned this in the PR description. 

To elaborate, our previous definition of an agent was:

> Agent: A System that interacts with an Environment. It can consume zero or more Observations, and perform zero or more Actions. The decision of which Actions to perform is made based on the Agent’s beliefs.

Which relied on the undefined term 'beliefs'.

Also, based on our conversation 2 meetings ago, we identified several stages on the spectrum between an Actor and an Agent (E.g. belief system, counterfactual depth, etc.), and we did not come to a consensus as to where we would draw the line. This PR basically sets the stage for a re-evaluation of the term Agent (Which is why I re-opened #28).